src/main/scala/strawman/collection/Set.scala

```diff
@@ -26,8 +27,81 @@ trait SetOps[A, +CC[X], +C <: Set[A]]
     */
   @`inline` final def apply(elem: A): Boolean = this.contains(elem)
 
+  /** Tests whether this set is a subset of another set.
+    *
+    *  @param that  the set to test.
+    *  @return     `true` if this set is a subset of `that`, i.e. if
+    *              every element of this set is also an element of `that`.
+    */
   def subsetOf(that: Set[A]): Boolean = this.forall(that)
 
+  /** An iterator over all subsets of this set of the given size.
+    *  If the requested size is impossible, an empty iterator is returned.
+    *
+    *  @param len  the size of the subsets.
+    *  @return     the iterator.
+    */
+  def subsets(len: Int): Iterator[C] = {
```

Comment:
These `subsets` operations return an `Iterator` in the current collections, so I did that also here. But what do you think of returning a `View` instead?